### DISCUSSION TITLE
Adding Challenge-Response Yubikey

### DISCUSSION THREAD

**Discussion by sidielnikov** [samuel-lucas6/Kryptor#38] [Ideas]
Hi samuel-lucas6. How to debur idea to add support for hardware keys Yubikey? As in KeePassxc.

**Comment by samuel-lucas6** [comment]
Hi @sidielnikov, I don't plan on implementing this currently. As I say [here](https://www.kryptor.co.uk/#out-of-scope) and [here](https://www.kryptor.co.uk/roadmap#suggesting-a-new-feature), very [few](https://latacora.micro.blog/2019/07/16/the-pgp-problem.html) people own YubiKeys, and I think it would be quite difficult to implement as I believe there are few libraries for it.
